cartoframes/vis/__init__.py

```diff
@@ -0,0 +1,37 @@
+"""
```

Comment:
I think `vis` or `visualization` is better here, because it can create confusion because the CARTO VL viz https://carto.com/developers/carto-vl/reference/#cartoviz.